components/images/fullpage-hero.css

```diff
@@ -52,10 +52,12 @@
   display: block;
   content: '⌄';
   font-size: var(--h2);
+  margin-top: 0.5rem;
```

Comment:
This only applies to the downward-facing arrow under the "read more" to compensate for the added padding